lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -181,7 +180,18 @@ func (ic *ICache) getImageByNamed(named reference.Named) (*metadata.ImageConfig,
 		}
 	}
 
-	return copyImageConfig(config), nil
+	return copyImageConfig(config)
+}
+
+// Add the "sha256:" prefix to the image ID if missing.
+// Don't assume the image id in image has "sha256:<id> as format.  We store it in
+// this format to make it easier to lookup by digest
+func prefixImageID(imageID string) string {
+	if strings.HasPrefix(imageID, "sha256:") {
+		return imageID
+	} else {
```

Comment:
golint agrees with @caglar10ur :

```
if block ends with a return statement, so drop this else and outdent its block
```

Our CI used to require our code is go-lint, go-vet, etc. clean, we really need to fix this.  @vburenin do you not have go-lint integrated with your editor?  Maybe you could take a look?